src/mount/mount.ceph.c

```diff
@@ -40,6 +40,8 @@ static char *mount_resolve_src(const char *orig_str)
 	char *mount_path;
 	char *src;
 	char *buf = strdup(orig_str);
+	if (!buf)
```

Comment:
i don't think the caller is expecting an errno. probably we can just print an error message and return `NULL` , like
```c++
fprintf(stderr, "%s: failed to allocate memory\n", __func__);
return NULL;
```